mysql-test/mroonga/storage/index/multiple_column/unique/t/decimal.test

```diff
@@ -30,6 +30,8 @@ select c1, c2, c3 from t1 force index(uk1) where c2 = -9876543210987654321098765
 select c1, c2, c3 from t1 force index(uk1) order by c2, c3;
 select c1, c2, c3 from t1 force index(uk1) order by c2 desc, c3 desc;
 select c2, c3 from t1 force index(uk1) order by c2, c3;
+#For MariaDB 10.2 or later
```

Comment:
```suggestion
# MariaDB 10.2 or later uses "..." to show the displayed value is truncated.
# We will use "..." version instead of "000" version as expected value
# when MySQL also uses "...".
```